tools/dynamatic/scripts/simulate.sh

```diff
@@ -69,9 +69,8 @@ echo_info "Launching Modelsim simulation"
 cd "$HLS_VERIFY_DIR"
 "$HLS_VERIFIER_BIN" \
```

Comment:
I also don't understand why most VHDL files are copied by `simulate.sh`, while the "supplementary" VHDL files are copied by `hls-verifier` (`copySupplementaryFiles` method).

In my view, `hls-verifier` should only run `compareCAndVhdlOutputs`. ModelSim script generation and testbench execution seem unrelated to HLS verification.